src/app/core/abstractions/entity-parser.service.ts

```diff
@@ -19,6 +19,7 @@ export abstract class EntityParser {
   public abstract parseShowStyleVariant(showStyleVariant: unknown): ShowStyleVariant
   public abstract parseStudioConfiguration(studioConfiguration: unknown): StudioConfiguration
   public abstract parseTv2Action(tv2Action: unknown): Tv2Action
-  public abstract parseMedia(media: unknown): Media
+  public abstract parseMediaAsset(media: unknown): Media
+  public abstract parseMediaAssets(media: unknown): Media[]
```

Comment:
Or the methods should be renamed to not include `assests` since we don't call them assets anywhere else.

I know that the methods then have the same name, but we could leverage method overloading for that.

Reply:
I have kept the `mediaAsset` approach for now. I think we need to figure out how we want this naming done in the code.